src/Pate/Discovery.hs

```diff
@@ -418,34 +436,27 @@ runDiscovery ::
   CME.ExceptT (PEE.EquivalenceError arch) IO ([Word64], PMC.BinaryContext arch bin)
 runDiscovery mCFGDir repr elf hints = do
   let archInfo = PLE.archInfo elf
-  entries <- F.toList <$> MBL.entryPoints bin
-
-  let (invalidHints, hintedEntries) = F.foldr (addFunctionEntryHints (Proxy @arch) mem) ([], entries) (PH.functionEntries hints)
-  let ds = MD.cfgFromAddrs archInfo mem Map.empty hintedEntries []
-
-  -- If the user wants to persist macaw CFGs, do so here
-  --
-  -- Save these CFGs to <dir>/<repr>/<address>
-  liftIO $ F.forM_ mCFGDir $ \cfgDir -> do
-    let baseDir = cfgDir </> show repr
-    SD.createDirectoryIfMissing True baseDir
-    F.forM_ (ds ^. MD.funInfo) $ \(Some dfi) -> do
-      let fname = baseDir </> show (MD.discoveredFunAddr dfi) <.> "cfg"
-      IO.withFile fname IO.WriteMode $ \hdl -> do
-        PPT.hPutDoc hdl (PP.pretty dfi)
```

Comment:
Need to restore this (probably at the incremental discovery stage - write out CFGs as we find them)